### ISSUE TITLE
OSM tile cache gets purged too frequently

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#9953]
Taken from https://github.com/cgeo/cgeo/issues/8149#issuecomment-599135469 by @jonas-koeritz:

c:geo is calling tileCache.purge() very often, effectively destroying its purpose. But removing the calls breaks the Mapsforge map...at least in my testing.

**Comment by jakedot** [comment]
Has there been any progress here during other developments?

**Comment by moving-bits** [comment]
No, and I'm not sure if there will ever be, as current map implementation is quite "grown" over the years, having many dependencies and interconnections.

But: We are currently working on a new map implementation, which tries to unify implementation for the different map types (Google Map, OSM Online, OSM Offline) as much as possible, reusing the business code and separating only the parts where the technical details differ. This `UnifiedMap` as we call it currently, will use a different caching mechanism for OSM online tiles, making this issue here obsolete. - Implementation is at an early stage, though, and will take many more months to be completed.

**Comment by moving-bits** [comment]
Won't fix. Old map implementations are deprecated as of release 2025.07.17 and superseded by UnifiedMap.
